source/_integrations/juicenet.markdown

```
@@ -22,8 +22,11 @@ ha_integration_type: integration
 
 The `juicenet` platform pulls data from a [JuiceNet](https://evcharging.enelx.com/products/juicebox) charging station equipped with a Wi-Fi connection. It will access and make available all of the devices attached to your account. It also exposes a switch allowing you to charge your car now instead of waiting for the pre-set schedule.
 
+> **⚠️ Warning:** This integration can no longer be set up as the required API token cannot be obtained. However, if you have it working, it may remain functional.
+
 {% include integrations/config_flow.md %}
 
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)